package.json

```diff
@@ -36,8 +36,7 @@
     "url": "https://github.com/stefan-prokop-cz/verify-apple-id-token/issues"
   },
   "dependencies": {
-    "jsonwebtoken": "^9.0.0",
-    "jwks-rsa": "^3.0.0"
+    "jose": "^4.14.4"
   },
   "devDependencies": {
     "@types/jest": "29.5.2",
```

Comment:
Can we please remove unused `@types/jsonwebtoken` package? :pray:

Reply:
All good now!